src/index.html

```diff
@@ -9,6 +9,40 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header_nav">
+      <a href="#home">
+        <img class="logo" src="./images/logo.png" alt="logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#Apple" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
```

Comment:
Don't forget to add empty lines between multiline sibling blocks of HTML. Fix it everywhere 
```suggestion
          </li>
          
          <li class="nav__item">
```